pkg/apis/batch/v1beta1/zz_generated.conversion.go

```diff
@@ -215,7 +214,8 @@ func Convert_batch_JobTemplate_To_v1beta1_JobTemplate(in *batch.JobTemplate, out
 
 func autoConvert_v1beta1_JobTemplateSpec_To_batch_JobTemplateSpec(in *v1beta1.JobTemplateSpec, out *batch.JobTemplateSpec, s conversion.Scope) error {
 	out.ObjectMeta = in.ObjectMeta
-	if err := batch_v1.Convert_v1_JobSpec_To_batch_JobSpec(&in.Spec, &out.Spec, s); err != nil {
+	// TODO: Inefficient conversion - can we improve it?
```

Comment:
Is this change intended? Looks weird.

Reply:
no, it's not.  It's weird that the conversion-gen changed it...